nilearn/plotting/displays.py

```diff
@@ -870,11 +950,17 @@ def annotate(self, left_right=True, positions=True, size=12, **kwargs):
         positions: boolean, optional
             If positions is True, annotations indicating the
             positions of the cuts are drawn.
+        scalebar: boolean, optional
+            If ``True``, cuts are annotated with a reference scale bar.
+            For finer control of the scale bar, please check out
+            :meth:`~nilearn.plotting.displays.BaseAxes.draw_scale_bar`.
         size: integer, optional
             The size of the text used.
         kwargs:
             Extra keyword arguments are passed to matplotlib's text
-            function.
+            function. If the keyword starts with `scale_`, then it is
+            redirected to :meth:`~nilearn.plotting.displays.BaseAxes.\
+            draw_scale_bar` if ``scalebar`` is ``True``.
```

Comment:
I am not sure how to say this: important argument should not be in kwargs. The function that you are contributing is not usable without understanding the kwargs. Understanding them requires careful reading of several docstrings. This is bad user experience.

An important goal of nilearn is to minimize cognitive overhead for users. This is why people like it: non experts can use it. Code that is contributed to nilearn should abide by these standard. I would say that the main goal of contributing code to nilearn should be to put it in the hand of non experts. Elsewhere code could live in a separate repository.

As I have said, the useful arguments should be passed as explicit arguments and not in kwargs.

I will not merge the code as such, as the goal of development in nilearn is to improvement usability at each new release.

Reply:
By the way, if this is helpful, I can implement this change, if I am not conveying my ideas clearly or if you don't have time.

Reply:
This got hidden from my email:

I just read the last comment about making the changes yourself. Of course,
go for it! I'd really appreciate that and will probably save time to both
of us.

Thanks very much